sourcemash/api/feeds.py

```diff
@@ -18,14 +20,75 @@
 
 class FeedListAPI(Resource):
```

Comment:
That's a good point. We might want that as an option in the future. Unsubscribe is the more common call though, so I think it makes sense to leave it as the DELETE to FeedList for now.

If we want to add an admin delete in the future, we can add a feeds/id/delete URL with role protection (admins-only) on it down the line.